README.md

```diff
@@ -15,39 +15,41 @@ Welcome to [Data Together](https://datatogether.org/), we're building a better f
 
 ## Get Involved
 
-If you would like to **add to the discussion**, please see our [Contributing Guidelines](./CONTRIBUTING.md) and [Code of Conduct](https://github.com/datatogether/datatogether/blob/master/CONDUCT.md). 
+If you would like to **add to the discussion**, please see our [Contributing Guidelines](./CONTRIBUTING.md) and [Code of Conduct](https://github.com/datatogether/datatogether/blob/master/CONDUCT.md).
 
 If you want to **get into code and start hacking**, check out [first timer](https://github.com/issues?q=is:open+org:datatogether+label:first-timers-only) issues! [Webapp](https://github.com/datatogether/webapp) and [sentry](http://github.com/datatogether/sentry) are the best points of entry. We track project progress using a [Waffle Board](https://waffle.io/datatogether/roadmap) and identify major goals in our [Roadmap repository](https://github.com/datatogether/roadmap/).
 
+If you would like to get **email notifications**, please join our [public google group](https://groups.google.com/forum/#!forum/datatogether/join)
+
 ### Community Calendar
 
 We have a public calendar that lists Data Together events and a Youtube playlists with recordings of our public calls!
 
-📅 [**Data Together Calendar**](https://calendar.google.com/calendar/embed?src=u75o4fbnv59006peo07nv67vsg%40group.calendar.google.com&ctz=America%2FToronto)  
-▶️ [**Call Playlist**](https://www.youtube.com/playlist?list=PLtsP3g9LafVul1gCctMYGm9sz5FUWr5bu)
+📅 [**Data Together Calendar**] (https://calendar.google.com/calendar/embed?src=u75o4fbnv59006peo07nv67vsg%40group.calendar.google.com&ctz=America%2FToronto)
```

Comment:
Please remove the space between `...Calendar**]` and `(https:...`
This breaks the format for markdown links and causes it appear not as a link anymore.
Here's a [cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links) for how to format links.

The spaces I suggested re-adding should be added at the end of the line like after `Toronto)`. Not in between the link text and url, which is what you did instead. Those all need to be fixed.

If you look at the diff view, in the section in red, it shows 2 spaces that were at the end of various line. They are there so that the links show up on different lines. The spaces were removed in your changes and should be added back.